hackathon_site/event/serializers.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from rest_framework import serializers
+
+from event.models import Profile
+
+User = get_user_model()
```

Comment:
If you're calling this again instead of importing it from the model won't it make two unconnected copies of the class

Reply:
Why do you mean "two unconnected copies of the class"? All `get_user_model()` does in our case is return `django.contrib.auth.models.User`, the default user model.

I could change the import to be `from event.models import Profile, User`, but it is functionally identical.

Reply:
Will update to use that import, since it's what the tests also do. It doesn't seem like the best practice since it's almost importing another import, but it makes things clearer and reduces some boilerplate.